src/testdir/test_functions.vim

```diff
@@ -253,22 +254,41 @@ func Test_resolve_win32()
     call delete('Xlink.lnk')
   endif
 
+  call delete('Xlink')
+  call delete('Xdir', 'd')
+  call delete('Xfile')
+
+  " test for symbolic link to a file
   new Xfile
   wq
-  silent !mklink Xlink Xfile > NUL
+  silent !mklink Xlink Xfile
   if v:shell_error
     " mklink require Administrator rights.
-     call delete('Xfile')
+    call delete('Xfile')
     return
   endif
   call assert_equal(s:normalize_fname(getcwd() . '\Xfile'), s:normalize_fname(resolve('./Xlink')))
   call delete('Xlink')
 
-  silent !mkdir Xdir
-  silent !mklink /D Xlink Xdir > NUL
+  " test for symbolic link to a directory
+  call mkdir('Xdir')
+  silent !mklink /D Xlink Xdir
+  if v:shell_error
+    " mklink require Administrator rights.
+    call delete('Xdir', 'd')
+    return
+  endif
+  call assert_equal(s:normalize_fname(getcwd() . '\Xdir'), s:normalize_fname(resolve(getcwd() . '/Xlink')))
+  call delete('Xfile')
+  call delete('Xdir', 'd')
+  call delete('Xlink', 'd')
+
+  " test for junktion link to a directory
```

Comment:
s/junktion link/junction/

Reply:
I don't think this has been resolved.
There are two mistakes.
1. Typo: 'k' -> 'c'
2. No need to say "junction link", "junction" is enough.